p4-16/spec/P4-16-spec.mdk

```diff
@@ -4394,6 +4396,39 @@ only if they have the same element type and the same length.  Two
 stacks are equal if and only if all their corresponding elements are
 equal.  Note that the `nextIndex` value is not used in the equality comparison.
 
+### Header stack expressions { #sec-hs-init }
+
+One can write expressions that evaluate to a header stack.  The
+syntax of these expressions is given by:
+
+~ Begin P4Grammar
+expression ...
```

Comment:
Minor nit that we can try to make more consistent with this PR, or a later one:

There are some cases where we do partial lists of grammar production rules like this:
```
expression
    : ...
    | expression '<' realTypeArgumentList '>' '(' argumentList ')'
    | expression '(' argumentList ')'
```
and others like `expression ...` as shown here, and one other variant I saw in another place.  I am happy to create a separate PR and make these all consistent, since that is not the main point of this PR.